### ISSUE TITLE
Feature Request: Allow -i <INTERFACE NAME> to show full output of selected interface

### ISSUE THREAD

**Issue by krosseyed** [jftuga/nics#4]
Wanted to see how difficult it would be to add `-i <NIC_NAME>` if I have a bunch of interfaces? I use MacOS and it shows a bunch of garbage info when I do -a, making it hard to parse and goes against the spirit of this application (IMHO).

This would also be used in conjunction with `-a` as it's more useful than doing `ifconfig en0` on MacOS

**Comment by krosseyed** [comment]
Thinking about this more, `-ai` is not needed and `-i` should be fine for this

**Comment by jftuga** [comment]
* The `-i` option is implemented in PR https://github.com/jftuga/nics/pull/5
* The `v1.5.0` binaries will include this feature

**Comment by krosseyed** [comment]
That works! Thanks!
